.cargo/config

```diff
@@ -0,0 +1,2 @@
+[build]
```

Comment:
It's better to add in CI only instead of changing everyoneis workflow.

Reply:
Yes, I still think which way is better...